### ISSUE TITLE
Strangley poor results on pre-trained model

### ISSUE THREAD

**Issue by DaddyWesker** [ThibaultGROUEIX/AtlasNet#75]
Hello and thanks for sharing your code.

I've got a problem. I've launched train.py --demo on your plane picture and it works fine, but i've tried to use my own pictures and got weirdly poor results. Probably I am missing something. Here is the examples:
![mustang_re](https://user-images.githubusercontent.com/10774222/175301961-2985b4e0-6fd1-4768-a08e-8622e6ae890c.png)
![image](https://user-images.githubusercontent.com/10774222/175302046-4a13cd65-08a6-48c2-bf55-d5f8a532acad.png)

![plane_new](https://user-images.githubusercontent.com/10774222/175302070-24f2c7b4-f911-494f-862a-b6ecc42adc87.png)
![image](https://user-images.githubusercontent.com/10774222/175302122-2a2076a2-5129-4110-a873-1d1153e16bf6.png)

I've used png images with black background (first pair), white background and without background (second pair) - same

**Comment by zergrisk1** [comment]
Have you removed the background?

**Comment by DaddyWesker** [comment]
Yes i did. I've downloaded files without background and even tried to create my own no background image. Still the same.

**Comment by zergrisk1** [comment]
Try to use the picture same size with the demo-plane picture?
Like the file below:
![00014](https://user-images.githubusercontent.com/80167158/182212292-3ee0a3de-08a4-4d28-883c-ce8dd4fce187.png)

**Comment by DaddyWesker** [comment]
@zergrisk1 

I've already tried with 137x137 images. For example this one

![cartoon_plane](https://user-images.githubusercontent.com/10774222/182301706-4bf14f72-b3ac-46af-9d26-468778bc7bc9.png)

gives

![image](https://user-images.githubusercontent.com/10774222/182301787-86e1805e-5d91-4a1a-9a96-7187ed494fda.png)

**Comment by DaddyWesker** [comment]
Your pic works fine. But here is mine

![Plane_2](https://user-images.githubusercontent.com/10774222/182303282-5f016a18-bfd4-4c86-9090-8cd56477ca09.png)

![image](https://user-images.githubusercontent.com/10774222/182303369-865f010d-3c74-4d1d-a30e-a79eae7cd4a6.png)

**Comment by kochsebastian** [comment]
@DaddyWesker were you able to resolve the issue? I am facing very similar problems.

**Comment by DaddyWesker** [comment]
@kochsebastian 
Unfortunately, no. On some pics model works decent, but on most pics i've used it works as i stated above.
